package.json

```
@@ -9,7 +9,7 @@
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "node --test",
```

can you add `c8` for coverage
```suggestion
    "test:unit": "c8 --100 node --test",
```